install/installer/cmd/root.go

```diff
@@ -23,11 +23,13 @@ func Execute() {
 }
 
 var rootOpts struct {
-	VersionMF string
+	VersionMF         string
+	StrictConfigParse bool
 }
 
 func init() {
 	rootCmd.PersistentFlags().StringVar(&rootOpts.VersionMF, "debug-version-file", "", "path to a version manifest - not intended for production use")
+	rootCmd.PersistentFlags().BoolVar(&rootOpts.StrictConfigParse, "strict-parse", true, "toggle strict configuration parsing")
```

Comment:
nit: being more explicit of what "strict" or "non-strict" means would be nice. E.g. I had to look up [the docs](https://pkg.go.dev/gopkg.in/yaml.v2#UnmarshalStrict) to learn that strict fails on a) unknown and b) _double_ fields.